src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -577,6 +581,54 @@ public boolean hasNoOngoingBusinessProcess() {
+    private final List<Element<UploadEvidenceExpert>> documentUploadExpert4;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial1;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial2;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial3;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial4;
+    private final LocalDateTime caseDocumentUploadDate;
+    //respondent
+    private final List<Element<UploadEvidenceDisclosure>> documentUploadDisclosure1Res;
+    private final List<Element<UploadEvidenceDisclosure>> documentUploadDisclosure2Res;
+    private final List<Element<UploadEvidenceWitness>> documentUploadWitness1Res;
+    private final List<Element<UploadEvidenceWitness>> documentUploadWitness2Res;
+    private final List<Element<UploadEvidenceWitness>> documentUploadWitness3Res;
+    private final List<Element<UploadEvidenceWitness>> documentUploadWitness4Res;
+    private final List<Element<UploadEvidenceExpert>> documentUploadExpert1Res;
+    private final List<Element<UploadEvidenceExpert>> documentUploadExpert2Res;
+    private final List<Element<UploadEvidenceExpert>> documentUploadExpert3Res;
+    private final List<Element<UploadEvidenceExpert>> documentUploadExpert4Res;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial1Res;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial2Res;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial3Res;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial4Res;
+    private final LocalDateTime caseDocumentUploadDateRes;
+    //respondent
+    private final List<Element<UploadEvidenceDisclosure>> documentUploadDisclosure1Other;
+    private final List<Element<UploadEvidenceDisclosure>> documentUploadDisclosure2Other;
+    private final List<Element<UploadEvidenceWitness>> documentUploadWitness1Other;
+    private final List<Element<UploadEvidenceWitness>> documentUploadWitness2Other;
+    private final List<Element<UploadEvidenceWitness>> documentUploadWitness3Other;
+    private final List<Element<UploadEvidenceWitness>> documentUploadWitness4Other;
+    private final List<Element<UploadEvidenceExpert>> documentUploadExpert1Other;
+    private final List<Element<UploadEvidenceExpert>> documentUploadExpert2Other;
+    private final List<Element<UploadEvidenceExpert>> documentUploadExpert3Other;
+    private final List<Element<UploadEvidenceExpert>> documentUploadExpert4Other;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial1Other;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial2Other;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial3Other;
+    private final List<Element<DocumentUploadTrial>> documentUploadTrial4Other;
+    private final LocalDateTime caseDocumentUploadDateOther;
```

Comment:
For other reviewers:
we are aware of the oddity around this whole structure. We'll be fixing this separately.